hotshot-state-prover/src/service.rs

```diff
@@ -103,62 +102,70 @@ pub fn init_stake_table(
     Ok(st)
 }
 
-async fn init_stake_table_from_orchestrator(
-    orchestrator_url: &Url,
+#[derive(Debug, Deserialize)]
+pub struct PublicHotShotConfig {
```

Comment:
Nit: why not just import this from sequencer?

Reply:
I tried that but cargo told me there was a cyclic dependency problem somewhere. I also considered moving it to `utils` but it didn't seem in the util spirit. 

I can keep pushing if we want though, maybe it wouldn't be too bad

Reply:
Deploying the light client contract will invoke `light_client_genesis` fn from `hotshot-state-prover` repo. Thus a cyclic dep.